sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -828,7 +830,9 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
         val execPlan = python.FlatMapGroupsInPandasWithStateExec(
           func, groupAttr, outputAttr, stateType, None, stateVersion, outputMode, timeout,
           batchTimestampMs = None, eventTimeWatermarkForLateEvents = None,
-          eventTimeWatermarkForEviction = None, planLater(child)
+          eventTimeWatermarkForEviction = None,
+          skipEmittingInitialStateKeys = false,
```

Comment:
This reminds me we do not support initial state in applyInPandasWithState, but it's OK since we are proposing a new feature which covers this.

Reply:
Right correct